app/configurator/config-types.ts

```diff
@@ -273,13 +281,16 @@ const AreaFields = t.intersection([
     ]),
   }),
 ]);
-const AreaConfig = t.type(
-  {
-    chartType: t.literal("area"),
-    filters: Filters,
-    interactiveFiltersConfig: InteractiveFiltersConfig,
-    fields: AreaFields,
-  },
+const AreaConfig = t.intersection(
+  [
+    t.type({
+      chartType: t.literal("area"),
+      filters: Filters,
+      interactiveFiltersConfig: InteractiveFiltersConfig,
+      fields: AreaFields,
+    }),
+    t.partial({ imputationType: ImputationType }),
+  ],
```

Comment:
Not sure is imputationType should be at this level; maybe it belongs to Y field? Or a segment field, as currently it's only used in conjunction with segments?

Reply:
Yeah … sounds more like a field-level config (could there be a case where x AND y are imputed?)? I don't think it just belongs to the segment part though.

Reply:
I think that generally, as of now, there isn't a case where x and y are both inputed. We are only imputing missing values that were "artificially" introduced to the data by selecting a particular chart type (stacked area chart) and not all the missing values. So let's say that we have an X temporal axis with a "yearly" resolution; if we are missing some years, we are not offering a way to impute these; only if a missing value appeared due to stacking of series of different timespans.

Reply:
To me, it would make sense to put it in the "y" field config. I do not have a super strong opinion though.